monitoring/mongodb/alerts.test.yaml

```diff
@@ -333,23 +333,31 @@ tests:
   - name: MongoDbRSNotSynced
     interval: 1m
     input_series:
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-0", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-1", member_state="SECONDARY"}
-        values: 0x10
-      - series: mongodb_rs_members_state{namespace="zenko", rs_nm="rs0", pod="data-db-mongodb-sharded-mongos-2", member_state="SECONDARY"}
+      - series: mongodb_rs_members_state{namespace="zenko", pod="data-db-mongodb-sharded-shard0-data-0", member_state="PRIMARY", rs_nm="data-db-mongodb-sharded-shard-0", rs_state="1",  member_idx="data-db-mongodb-sharded-shard0-data-0.data-db-mongodb-sharded-headless.zenko.svc.cluster.local:27017"}
```

Comment:
member_idx has been added as it influences the calculous , without it we get a result of 1 , but once it is provided ( matching the real life metrics within the plateforms ) the calculous becomes 2 matching the right number of secondaries in the replicaset / pod

Reply:
it does not _influence the calculations_ : only if you don't include it, you actually provide the exact same time-series multiple times and you end up actually having a single serie.... hence the result of the computation.